src/server/verification/verificationAPI.js

```diff
@@ -855,6 +856,24 @@ const setup = (app: Router, verifier: VerificationAPI, gunPublic: StorageAPI, st
       })
     })
   )
+
+  /**
+   * @api {get} /verify/phase get release/phase version number
+   * @apiName Get Phase VErsion Number
+   * @apiGroup Verification
+   *
+   * @apiSuccess {Number} phase
+   * @apiSuccess {Boolean} success
+   * @ignore
+   */
+  app.get(
+    '/verify/phase',
+    passport.authenticate('jwt', { session: false }),
```

Comment:
1. we don't need authentication here
2. i think we don't need wrapAsync (and declare route function as async) as we don't have any async code or possible exceptions to be thrown
3. you don't use req and next so change function declaration to (_, res) => {
4. just to keep coverage level please add very simple unit tests which calls /verify/phase _without_ creds and checks response for { success: true, phase: phase value from config)